File: Discrete_Mathematics/Fundamentals OF Mathematical Logics/PYTHON Implementations/disjunction.py
# Function to implement Disjunction

def check_disjunction(p, q):
    if (p or q) == True:
        return True
    else:
        return False

result = check_disjunction(False, True)
print(result)  # output true

result = check_disjunction(True, False)
print(result)  # output true

result = check_disjunction(True, True)
print(result)  # output true

result = check_disjunction(False, False)
print(result)  # output false

File: Discrete_Mathematics/Fundamentals OF Mathematical Logics/Java Implementation/Disjunction.java
// Java Implementation of Disjunction Propositions

public class Disjunction {

    // Function to check proposition
    public boolean IsDisjunction(boolean p, boolean q){
        if(p || q == true){
            return true;
        }
        else{
            return false;
        }
    }

    // tests
    boolean output;
    public void main(String[] args) {
        output =IsDisjunction(true, false);
        System.out.println(output); // output: true

        output =IsDisjunction(true, true);
        System.out.println(output); // output: true

        output =IsDisjunction(false, false);
        System.out.println(output); // output: false

        output =IsDisjunction(false, true);
        System.out.println(output); // output: true
    }
}
